compiler/bindings/python/iree/compiler/tools/import_onnx/importer_externalization_overrides.py

```diff
@@ -155,6 +175,20 @@ def define_function(
         # Recover per-module caches of various attributes.
         # Allows modification in the same module_op without
         # loss of current state.
+        all_input_map = copy.deepcopy(graph_info.input_map)
+        num_inputs = len(all_input_map.items())
+        if (
+            param_data.input_index_threshold is not None
+            and param_data.input_index_threshold < num_inputs
+        ):
+            thresh = (
+                param_data.input_index_threshold
+                if param_data.input_index_threshold >= 0
+                else max(0, param_data.input_index_threshold + num_inputs)
+            )
+            for _index in range(thresh, num_inputs):
+                _discarded = graph_info.input_map.popitem()
```

Comment:
nit: spell out 'threshold'?

Actually, can you document what happens if `input_index_threshold` is negative? What is the intent here?

Reply:
I just wanted to avoid crashing if someone tried like `input_index_threshold=-1` when trying to externalize the last input as a param.

Reply:
So it will try to shift negative input indices by the number of inputs, and if it's still negative, then it clips to 0.

Reply:
Could you instead raise an exception if the threshold is negative, rather than ignore the invalid input?